### PR TITLE
Updated all the files's req.env ==> req.variables.

### PR DESCRIPTION
**Pull request by 35C4n0r**
I've updated all the files in which ```req.env``` was being used previously to ```req.variables```. I've also remove emojis from some of the files and comments.
Signed-off-by: 35C4n0r <jaykumar20march@gmail.com>

### CODE REVIEW

**Review by Meldiron — CHANGES_REQUESTED** [review]
Hey there 👋 Thanks for the PR. I noticed you missed a few files in PHP:
- object_detection
- generate_giphy_gif
- convert-phone-number-to-country-name

Please update those too, same thing, renaming from `env` to `variables`.

**Comment by 35C4n0r** [comment]
> Hey there 👋 Thanks for the PR. I noticed you missed a few files in PHP:
> 
> * object_detection
> * generate_giphy_gif
> * convert-phone-number-to-country-name
> 
> Please update those too, same thing, renaming from `env` to `variables`.

@Meldiron thanks for the review, i've made the changes.

**Review by Meldiron — APPROVED** [review]
Looks great 😍

**Comment by thinkverse** [comment]
> Hey there 👋 Thanks for the PR. I noticed you missed a few files in PHP:
> 
>     * object_detection
> 
>     * generate_giphy_gif
> 
>     * convert-phone-number-to-country-name

I already have two PRs for updating the Giphy (https://github.com/open-runtimes/examples/pull/74) and Object Detection (https://github.com/open-runtimes/examples/pull/75) once. Also, shouldn't the docker runtimes also be updated to `v2` as well?

**Comment by 35C4n0r** [comment]
> I already have two PRs for updating the Giphy (#74) and Object Detection (#75) once. Also, shouldn't the docker runtimes also be updated to `v2` as well?

@thinkverse for updating docker runtimes to ```v2``` we only need to change the commands provided in the READMEs, right?

**Comment by thinkverse** [comment]
> updating docker runtimes to `v2` we only need to change the commands provided in the READMEs

That's my assumption yes, @Meldiron could confirm. Since it's the `v2` image tags that use `variables` instead of `env`.

**Comment by stnguyen90** [comment]
@35C4n0r, yes please update the readmes except for the ones that would conflict with #74 and #75 🙏🏼

**Comment by 35C4n0r** [comment]
@stnguyen90 @Meldiron There was a conflict, which has now been resolved. Please review and merge this PR.

**Review by Meldiron — APPROVED** [review]

**Review by christyjacob4 — APPROVED** [review]

File: php/generate_giphy_gif/index.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
     ) {
         throw new \Exception('Please provide all required environment variables.');
     }
-
+
```
Comment (thinkverse) [inline php/generate_giphy_gif/index.php:24]:
```suggestion

```

Remove this leftover space. 👍

**Comment by Meldiron** [comment]
Thanks for the amazing PR 😇
Merging the PR now 🥳

**Pull request merged**